core/src/main/java/com/klaviyo/coresdk/networking/KlaviyoNetworking.kt

```diff
@@ -0,0 +1,26 @@
+package com.klaviyo.coresdk.networking
+
+import com.klaviyo.coresdk.KlaviyoConfig
+import com.klaviyo.coresdk.networking.requests.IdentifyRequest
+import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
+import com.klaviyo.coresdk.networking.requests.TrackRequest
+
+fun track(event: KlaviyoEvent, customerProperties: Map<String, String>, properties: Map<String, String>? = null) {
```

Comment:
New top-level functions that help hide more of the SDK's implementation

Reply:
i think it would be nicer to wrap this in a class of some sort like:
```
object KlaviyoEvents {
    fun track(...
}
```
this way, developers implementing will do KlaviyoEvents.track(...), of course, not sure how this would work in a Java based project...

Reply:
I kinda thought the opposite would be better. I was fond of the idea of having top-level function calls for track and identify as it's a cleaner API.
I spose the drawback is that the naming is generic enough that there could be overlap with other dependencies leading to some developer confusion.

Reply:
Also calling Kotlin object methods in Java is just the same as you would in Kotlin. top-level functions would need to invoke the Kotlin file of origin to work through Java, and cleaning that up to be more Java-esque would have to be included in the Java wrapper for this SDK